pkg/registry/core/shoot/storage/tableconvertor.go

```diff
@@ -40,16 +41,20 @@ func newTableConvertor() rest.TableConvertor {
 		headers: []metav1beta1.TableColumnDefinition{
 			{Name: "Name", Type: "string", Format: "name", Description: swaggerMetadataDescriptions["name"]},
 			{Name: "Provider", Type: "string", Format: "name", Description: swaggerMetadataDescriptions["provider"]},
-			{Name: "CloudProfile", Type: "string", Format: "name", Description: swaggerMetadataDescriptions["cloudprofile"]},
+			{Name: "Region", Type: "string", Format: "name", Description: swaggerMetadataDescriptions["region"]},
 			{Name: "Seed", Type: "string", Format: "name", Description: swaggerMetadataDescriptions["seed"]},
-			{Name: "Version", Type: "string", Format: "name", Description: swaggerMetadataDescriptions["version"]},
+			{Name: "K8S Version", Type: "string", Format: "name", Description: swaggerMetadataDescriptions["k8sVersion"]},
 			{Name: "Hibernation", Type: "string", Format: "name", Description: swaggerMetadataDescriptions["hibernation"]},
 			{Name: "Last Operation", Type: "string", Format: "name", Description: swaggerMetadataDescriptions["lastoperation"]},
-			{Name: "APIServer", Type: "string", Format: "name", Description: swaggerMetadataDescriptions["apiserver"]},
-			{Name: "Control", Type: "string", Format: "name", Description: swaggerMetadataDescriptions["control"]},
-			{Name: "Nodes", Type: "string", Format: "name", Description: swaggerMetadataDescriptions["nodes"]},
-			{Name: "System", Type: "string", Format: "name", Description: swaggerMetadataDescriptions["system"]},
+			{Name: "Healthy", Type: "string", Format: "name", Description: swaggerMetadataDescriptions["healthy"]},
 			{Name: "Age", Type: "date", Description: swaggerMetadataDescriptions["creationTimestamp"]},
+			{Name: "CloudProfile", Type: "string", Priority: 1, Format: "name", Description: swaggerMetadataDescriptions["cloudprofile"]},
```

Comment:
I would probably vote for keeping the `CloudProfile` column in the default view instead of `Provider` and only add both `CloudProfile` and `Provider` in the wide view, as there can be multiple CloudProfiles per provider type and the CloudProfile (i.e. specific infrastructure) might generally be more interesting to operators/endusers.

Reply:
Thanks!
You are right, cloud profile name is also important. But I will keep `PROVIDER` as mandatory as the cloud profile name can be any valid k8s resource name, e.t. `xyz` and this alone might not help much. 

The `SEED` however should not matter so much in environment where multiple similar seeds are used, therefore I am making this column optional.